examples/paraview/Dockerfile

```diff
@@ -21,10 +22,10 @@ RUN pip3 install --no-binary=mpi4py \
 
 # Mesa. We need a version newer than Debian provides. Not sure which bug fix
 # resolved it but versions prior to 19 appear to have issues rendering on ARM.
```

Comment:
1. CentOS now, right?
2. Do we need to build it manually still?

Reply:
Corrected. We need 19+ which is newer than what is available to work around issues on ARM. I adjusted the comment to be more clear